### PR TITLE
Integral half life

### PR DESCRIPTION
**Pull request by nerdCopter**

### CODE REVIEW

File: src/main/cms/cms_menu_imu.c [hunk 1]
```diff
@@ -126,16 +140,108 @@ static long cmsx_rateProfileIndexOnChange(displayPort_t *displayPort, const void
...
+
+    pidProfile->feathered_pids = feathered_pids;
+    pidProfile->errorBoost = errorBoost;
+    pidProfile->errorBoostLimit = errorBoostLimit;
+    pidProfile->errorBoostYaw = errorBoostYaw;
+    pidProfile->errorBoostLimitYaw = errorBoostLimitYaw;
+    pidProfile->i_decay = i_decay;
+    pidProfile->integral_half_life = integralHalfLife;
+    pidProfile->integral_half_life_yaw = integralHalfLifeYaw;
+    pidProfile->QuickFlashRelax = QuickFlashRelax;
```
Comment (gretel) [inline src/main/cms/cms_menu_imu.c:189]: uhm?

File: src/main/flight/pid.h [hunk 2]
```diff
@@ -69,6 +63,14 @@ typedef enum {
     PID_CRASH_RECOVERY_BEEP
 } pidCrashRecovery_e;
 
+typedef enum {
+    HARDFLEX = 0, //error
```
Comment (gretel) [inline src/main/flight/pid.h:67]: nah.

File: src/main/cms/cms_menu_imu.c (hunk 1 above)
Comment (Quick-Flash) [reply src/main/cms/cms_menu_imu.c:189]: naming was off on purpose to help prevent testers from seeing what exactly they are testing for less placebo.

File: src/main/flight/pid.h (hunk 2 above)
Comment (Quick-Flash) [reply src/main/flight/pid.h:67]: hardflex is iterm relax based on pid error, unicorn is based on setpoint, and juicy is based on gyro. I don't see how that is misleading???

File: src/main/cms/cms_menu_imu.c (hunk 1 above)
Comment (gretel) [reply src/main/cms/cms_menu_imu.c:189]: the naming convention of variables is not related to their purpose.

File: src/main/flight/pid.h (hunk 2 above)
Comment (gretel) [reply src/main/flight/pid.h:67]: as long as you guys maintain the code it might be funny.

File: src/main/cms/cms_menu_imu.c (hunk 1 above)
Comment (nerdCopter) [reply src/main/cms/cms_menu_imu.c:189]: it is only temporary naming for the purpose of "blind testing".

File: src/main/flight/pid.h (hunk 2 above)
Comment (nerdCopter) [reply src/main/flight/pid.h:67]: * it is only temporary naming for the purpose of "blind testing".
* not intended to merge as-is.

File: src/main/cms/cms_menu_imu.c (hunk 1 above)
Comment (gretel) [reply src/main/cms/cms_menu_imu.c:189]: so the testers read the source code in the OSD?

File: src/main/flight/pid.h (hunk 2 above)
Comment (gretel) [reply src/main/flight/pid.h:67]: @nerdCopter pull requests have a purpose. they are made to be merged. or at least commented on by the community. why dont you work on branches and keep off PRs when you want to have that placebo thing going?

**Pull request closed without merging**